docs/SqlBindingRelatedRepos.md

```diff
@@ -28,9 +28,9 @@ Contains templates for the various types of bindings and supported languages, ea
 
 ## Extension Bundle
 
-Currently SQL Bindings is part of the 4.x bundles.
+Currently SQL Bindings is part of the 3.x and 4.x Preview bundles.
```

Comment:
Make a task to follow up correcting/updating this (we're not only in the preview bundles anymore)